pkg/cableengine/ipsec/strongswan.go

```diff
@@ -286,6 +296,43 @@ func (i *strongSwan) GetActiveConnections(clusterID string) ([]string, error) {
 	return connections, nil
 }
 
+// removeStaleCable removes cables that have no active SAS and returns a
+// boolean indicating removal of the cable. The status of the error is nil
+// in case of no errors, otherwise the returns values are false and the
+// error.
+func (i *strongSwan) removeStaleCable(client *goStrongswanVici.ClientConn, cableID string) (bool, error) {
+	sas, err := client.ListSas("", "")
+	if err != nil {
+		klog.Errorf("error while retrieving active sas")
```

Comment:
We don't need to log here are as we're propagating the error ...

Reply:
Yup, I agree with you on this. I left it like this because even though the error is propagated, there is no actual logging of it higher in the stack (IIRC). And since I didn't want to refactor those parts now I erred on logging and propagating.